script.js

```diff
@@ -36,10 +35,6 @@ function addBookToLibrary(book) {
   
 }
 
-
-
-
-
 let carrier;
 
 let bookCollection=document.querySelector('.bookCollection');
```

Comment:
place carrier and bookCollection on top of the file if not needed to be here

Reply:
done